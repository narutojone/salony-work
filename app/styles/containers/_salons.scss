.image {
    width: 50px;
    height: 50px;
}

.margintop {
    margin-top: 50px;
}

.secondtable th {
    display: none;
}

.lazyother {
    margin-top: -21px;
}

.lazysecond {
    margin-top: -30px;
}