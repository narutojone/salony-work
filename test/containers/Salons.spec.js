
import React from 'react';
import { shallow } from 'enzyme';
import { Salons } from 'containers/Salons';

const mockDispatch = jest.fn();

const props = {
  app: {
    alerts: [],
  },
  dispatch: mockDispatch,
  user: {
    isAuthenticated: false,
  },
};

function setup(ownProps = props) {
  return shallow(
    <Salons {...ownProps} />,
    { attachTo: document.getElementById('react') }
  );
}

describe('Salons', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should have dispatched an action on mount', () => {
    expect(mockDispatch.mock.calls[0][0]).toEqual({
      payload: { query: 1 },
      type: 'GET_PRODUCT',
    });
  });

  it('should render some items when data arrives', () => {
    wrapper.setProps({
      app: {
        app: {
          data: {
            products: {
              images: [],
            },
          },
        },
      },
    });
    expect(wrapper.find('.container')).toMatchSnapshot();
  });

  it('should have dispatched an action on mount', () => {
    expect(mockDispatch.mock.calls[0][0]).toEqual({
      payload: { query: 1 },
      type: 'GET_PRODUCT',
    });
  });
});
