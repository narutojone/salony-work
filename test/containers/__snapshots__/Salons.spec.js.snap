// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Salons should render some items when data arrives 1`] = `null`;
